### DISCUSSION TITLE
vCPU or CPU in benchmarking (https://docs.jans.io/v1.0.12/admin/recipes/benchmark/)

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4789] [Q&A] [answered]
Hello @ossdhaval,

The tables in the Benchmarking contains CPU. I need to know if they are **vCPU** or **CPU**. If CPU can I please know the specification of the CPUs used.

**Comment by ossdhaval** [comment, marked as answer]
That is vCPU. We will fix the document. Thanks for bringing this up :+1:

**Comment by ossdhaval** [comment]
Hi @dhanan-np Can you confirm if the above comment answers your question? If so, please accept it as an answer, and we can close this discussion. We are tracking action item for this using GH issue as noted in `Events` section.
